src/Stripe.net/Entities/PaymentSource.cs

```diff
@@ -3,7 +3,7 @@
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
-    public enum SourceType
+    public enum PaymentSourceType
     {
         Account,
```

Comment:
Not a blocker for this PR, but we talked about adding an `Unknown` value in the enum, to avoid defaulting to `Account` in case the API returns a type that the library doesn't know about.